test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -258,7 +259,11 @@ public void testStandbyTransactionLogging() throws Exception {
     private void subscribe(String mapName) {
         // Subscribe to mapName and upon changes stop standby LR
         CorfuStore corfuStore = new CorfuStore(standbyRuntime);
-        CorfuStoreMetadata.Timestamp ts = corfuStore.getTimestamp();
+        Token token = standbyRuntime.getSequencerView().query().getToken();
+        CorfuStoreMetadata.Timestamp ts = CorfuStoreMetadata.Timestamp.newBuilder()
+                .setEpoch(token.getEpoch())
+                .setSequence(token.getSequence())
+                .build();
```

Comment:
is queryToken necessary here? NON_ADDRESS seems to be ok.

Reply:
Also given that we have at least one use case where we need to subscribe from the latest, should we overload the subscribeListener() api to not have a timestamp, where we will under the hood subscribe to the latest timestamp?

Reply:
I already added that API and actually added one more where the table names are also not needed. Changed this code to subscribe to latest timestamp instead of using queryToken.